### ISSUE TITLE
java.lang.RuntimeException: There is already a dimension with this id

### ISSUE THREAD

**Issue by Sunekaer** [McJty/RFToolsDimensions#336]
Issue description: 
User crashed when making dimension

Versions:
* Minecraft: 1.16.5
* Forge: 36.0.45
* McJtyLib: 5.0.17
* RFTools: rftoolsbase-1.16-2.0.10
* RFTools Dimensions:  1.16-7.0.4-beta

https://paste.feed-the-beast.com/view/raw/f7811a6f

**Comment by McJty** [comment]
How did you do this exactly? Steps in full detail

**Comment by Sunekaer** [comment]
This is currently all i have from the user "crash when making a dimension with rftool dimension when using dimension builder".
Ill ask for full steps by steps

**Comment by billyma6** [comment]
I've obtained this crash as well. What I did was:
1. Create a dimension using a dimension tab; dimlet content and name doesn't matter
2. Fully build the dimension and load it with power
3. Create another dimension in another dimension tab, regardless of dimlet content, but keep the same name as the previous dimension
4. build the second dimension tab fully

When the second dimension reaches 99% or higher (so basically completion) the game will freeze and crash. The error log says what's in the title: "There is already a dimension with this id"
